src/usps_processor/db.clj

```diff
@@ -1,7 +1,8 @@
 (ns usps-processor.db
-  (:require [turbovote.datomic-toolbox :as d]))
+  (:require [turbovote.datomic-toolbox :as d]
```

Comment:
I agree. How do you feel about this convention?

`datomic` for `datomic.api`
`datomic-tb` for `turbovote.datomic-toolbox`